app/models/shareconomy/package.rb

```diff
@@ -0,0 +1,8 @@
+module Shareconomy
+  class Package < ActiveRecord::Base
+    belongs_to :listing
+    has_many :orders
+
```

Comment:
Why do we need package? @GeorgeZhukov, @kolyastepanets, could you please provide a final model's structure first?